CHANGELOG.md

```diff
@@ -5,6 +5,15 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/), and this project adheres to
 [Semantic Versioning](http://semver.org).
 
+## v1.1.0 - 2018-09-25
+
+### Changed
+- Update `hero-openmp-examples` to `v1.2.0`
```

Comment:
I think the Changelog is more valuable to everyone reading it if a brief summary of the changes of the version update is given.
For example:
```
- Update `hero-openmp-examples` to `v1.2.0`, enabling extensible (double-colon) `run` and `clean`
  Makefile rules and readding the inclusion of `omp.h`.
- Update `libhero-target` to `v1.1.0`, fixing the `omp.h` inclusion in the heterogeneous application
  (#22).
```